unit/path_strategies.cpp

```diff
@@ -334,6 +373,11 @@ void _check_with_strategy(
   safety_checkert::resultt result = bmc.run(gm);
   symex_eventt::validate_result(events, result);
 
+  if(
+    result == safety_checkert::resultt::UNSAFE &&
+    opts.get_bool_option("stop-on-fail") && opts.is_set("paths"))
+    worklist->clear();
```

Comment:
Nit pick: could you please add braces around `worklist->clear();` as multi-line conditions otherwise make the code hard to read. (Applies below as well.)